### PR TITLE
feat: build latest debian against bullseye

### LINKED ISSUE #51: Debian images built against buster, not bullseye
I see the bullseye images haven't been updated in a year while buster images still do. Bullseye has already gone stable and should probably replace the Buster images that are starting to contain fairly out of date software. For example, if I want to build a Python based piece of software on top of the image, there are no images now that contain even Python3.8, I'm stuck with Python3.7 that shipped with Buster in 2019.

### PR DESCRIPTION
**Pull request by Artanicus**
Fixes #51

### CODE REVIEW

**Comment by hertzg** [comment]
Thanks for the PR, have you tested building the docker image and that it still runs on `buster` ?

**Comment by hertzg** [comment]
It would be great if you could help with tweaking the pipeline on forks, but for now you can use the docker compose to build it locally (with or without `buildx`).

Something like this would build it for your cpu arch at least and create 
```
$ docker-compose build master --build-arg debianVersion=buster --build-arg rtl433GitRevision=master
```
you can play around with the `rtl433GitRevision=master` build-arg as well.

You will end up with an image with tag `hertzg/rtl_433:deban-master` on your local machine and you can try running it to test if it still works.

References:

* https://github.com/hertzg/rtl_433_docker/blob/36a056dfe5b788f17f44cd8d4746e679d7428242/images/debian/docker-compose.yaml#L9

* https://docs.docker.com/compose/reference/build/

**Comment by Artanicus** [comment]
After some fixes to the compose file (`deban` -> `debian` and added pathing to the Dockerfile) I can get fairly far into the build with the command `docker-compose -f images/debian/docker-compose.yaml build --build-arg debianVersion=buster --build-arg rtl433GitRevision=master master`, but it fails with:
```
Step 5/22 : ADD ./rtl_433 ./rtl_433
ERROR: Service 'master' failed to build: ADD failed: file not found in build context or excluded by .dockerignore: stat rtl_433: file does not exist
```

This is regardless of buildargs, so looks like your build system just doesn't work standalone. Perhaps it's easier if you create a PR of your own with the same tiny change, and then GitHub will trigger all the javascript whizbangs correctly. :-)

**Comment by Artanicus** [comment]
Ok, I figured out the missing link, just had to manually clone merbanan/rtl_433 into the top dir. Initial testing says there's some minor tweaks needed, for example bumping libsoapysdr to 0.7, but now it builds. Will test more thoroughly and add the tweaks.

File: images/debian/docker-compose.yaml [hunk 1]
```diff
@@ -2,9 +2,10 @@ version: '3'
 
 services:
   master:
-    image: hertzg/rtl_433:deban-master
+    image: hertzg/rtl_433:debian-master
     build:
       context: ../..
+      dockerfile: images/debian/build-context/Dockerfile
```
Comment (hertzg) [inline images/debian/docker-compose.yaml:8]: If you are making these paths relative to the root of the project then you should update all of the other docker-compose files to the same approach, consistency.

I'd leave this to be relative to the path this file is in which would be just `Dockerfile` which is the default.

**Review by hertzg — COMMENTED** [review]
Other than the consistency comment, lgtm

**Comment by hertzg** [comment]
I've updated the checks to skip docker login for what I think is for forks, if you could also update the branch from master we could try running the checks on the PR as well. 🙌

**Comment by Artanicus** [comment]
Rebased from master so now they may succeed. :-)

**Review by hertzg — APPROVED** [review]

**Comment by hertzg** [comment]
Ok, I think now it should skip the login, let's wait for the checks to succeed and then merge :shipit:. 
Thanks for the PR and helping with getting pipeline working with forks 🙌

**Comment by hertzg** [comment]
Release: https://github.com/hertzg/rtl_433_docker/runs/4967457755

**Comment by Artanicus** [comment]
Looking good, thanks!

**Pull request merged**